main.js

```diff
@@ -37,4 +37,60 @@ for (i = 0; i < arr3.length; i++) {
     if (arr3[i] % 2 == 0 && arr3[i] != 0) {
         console.log(arr3[i])
     }
-}
\ No newline at end of file
+}
+
+//lesson-3--1
+function palindrome(word) {
+    let arr = word.split('').reverse().join('');
+    if (word === arr) {
+        return 'true';
+    }
+    else {
+        return 'false';
+    }
+}
+console.log(palindrome('Ilya'));
+
+//3--2
+function min(a, b) {
+    if (a < b) {
```

Comment:
use `?` operator for this, an example on the 66 line.

Reply:
i don't understand what should I do. On the 66 line this alredy done.